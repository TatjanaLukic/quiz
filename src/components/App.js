import React from "react";
import JavaScriptQuiz from "./quizes/JavaScriptQuiz";
import ReactQuiz from "./quizes/ReactQuiz";
import CSSQuiz from "./quizes/CSSQuiz";
import Home from './Home';
import { HashRouter, Route, Switch } from "react-router-dom";
import Navbar from "./Navbar";

const App = () => {
  return (
    <HashRouter>
      <Navbar />
      <Switch>
        <Route exact path="/">
          <Home />
        </Route>
        <Route path="/JavaScriptQuiz">
          <JavaScriptQuiz />
        </Route>
        <Route path="/ReactQuiz">
          <ReactQuiz />
        </Route>
        <Route path="/CSSQuiz">
          <CSSQuiz />
        </Route>
      </Switch>
    </HashRouter>
  );
};

export default App;
