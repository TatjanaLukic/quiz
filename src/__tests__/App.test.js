import React from 'react';

import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import App from '../components/App';

configure({adapter: new Adapter()});

describe('<App/>',() => {

it("spanshot matches", () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});