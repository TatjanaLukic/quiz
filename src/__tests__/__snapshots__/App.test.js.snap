// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> spanshot matches 1`] = `
<HashRouter>
  <Navbar />
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Home />
    </Route>
    <Route
      path="/JavaScriptQuiz"
    >
      <JavaScriptQuiz />
    </Route>
    <Route
      path="/ReactQuiz"
    >
      <ReactQuiz />
    </Route>
    <Route
      path="/CSSQuiz"
    >
      <CSSQuiz />
    </Route>
    <Redirect
      to="/"
    />
  </Switch>
</HashRouter>
`;
